share/db_data.dba

```diff
@@ -23048,16 +23048,16 @@ keywords=cms, publish, e-commerce, content management, development framework',
         0 => '10',
         1 => 'nospam@ez.no',
         2 => 'anonymous',
-        3 => '4e6f6184135228ccd45f8233d72a0363',
-        4 => '2',
+        3 => '$2y$10$wUrvpYTs9KZ0mtB9Gzr.G.nk6wGhPYzcH3jl9Tk5DtxJN8povoQi2',
+        4 => '6',
```

Comment:
I'd like to see this consistent with the default value in config (`php_default`)

Reply:
@emodric @andrerom Ok. Do you know which password should be used for the anonymous hash? If it matters at all... :)

Reply:
IIRC, I generated those hashes for `publish` password, for all accounts.

Reply:
Nope, I checked and that's not it. But I can as well use it, I don't think it matters what it is.

Reply:
You're right. It's `anonymous` for anonymous user and `publish` for admin user.

Reply:
That doesn't fit the existing data either. I used `publish` for both.